.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -42,9 +42,12 @@ Why should this be added to the codebase?
 #### Reproduces how often:
 <!-- What percentage of the time does it reproduce? -->
 
+#### Platform and environment:
+<!-- OS, Java version, browser, etc, as applicable -->
+
 ### Version
```

Comment:
✏️ `Affects versions` may be more appropriate since someone may write a bug against multiple versions.